requirements/edx/base.txt

```diff
@@ -71,6 +71,7 @@ path.py==7.2
 piexif==1.0.2
 Pillow==3.1.1
 polib==1.0.3
+py2neo==3.1.1
```

Comment:
nit: I wonder if this should move to the bottom of the file, pointing out that it's needed for modulestore analysis or something like that.